### PR TITLE
WIP - Add test for cljs.spec.test.alpha/check

### PR DESCRIPTION
**Pull request by arichiardi**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -28,7 +28,14 @@ script:
 - yarn test
 - boot test
 - boot release-ci
-- ./build/lumo -c src/test/cljs:src/test/lumo:src/test/cljs_cp scripts/lumo_test.cljs --test-cli-option true
+- boot -qPB with-cp --write --file cp |
```
Comment (anmonteiro) [inline .travis.yml]: this is not needed

File: .travis.yml [hunk 2]
```diff
@@ -28,7 +28,14 @@ script:
...
+     -d org.clojure/spec.alpha:0.1.123
+     -d org.clojure/test.check:0.10.0-alpha2
+- ./build/lumo -c src/test/cljs:src/test/lumo:src/test/cljs_cp |
+     -D org.clojure/spec.alpha:0.1.123
```
Comment (anmonteiro) [inline .travis.yml]: why do we need the spec JAR?

File: .travis.yml [hunk 3]
```diff
@@ -28,7 +28,14 @@ script:
...
+     -D org.clojure/test.check:0.10.0-alpha2
```
Comment (anmonteiro) [inline .travis.yml]: `-L ./third_party` while we use a custom CLJS version

File: .travis.yml (hunk 1 above)
Comment (arichiardi) [reply .travis.yml]: I figured we needed a way to resolve the dependencies to `.m2`, so this is one way, we could use `mvn` as well I guess.

File: src/test/lumo/lumo/build_api_tests.cljs [hunk 4]
```diff
@@ -337,37 +337,52 @@
...
+                                   (slurp (path/join out "emit_node_requires_test/native_modules.js"))))))
       (is (empty? @ws))))
   (test/delete-node-modules))
 
-(deftest cljs-test-compilation
-
-  (testing "success"
-    (let [out (path/join (test/tmp-dir) "compilation-test-out")
-          root "src/test/cljs_build"]
+#_(deftest cljs-test-compilation
```
Comment (anmonteiro) [inline src/test/lumo/lumo/build_api_tests.cljs]: Why is this commented out?

Comment (arichiardi) [reply src/test/lumo/lumo/build_api_tests.cljs]: Sorry it is wip, I will put things back where they were

**Pull request merged**